src/components/form.RadioGroup/form.RadioGroup.tsx

```diff
@@ -0,0 +1,44 @@
+import { Form } from "../form";
+import { useForm } from 'react-hook-form';
+
+type FormData = {
+    diabetesType: string;
+};
+
+
+type DiabetesFormProps = {
+    title: string;
+    description: string;
+    options: string[];
+}
+
+const DiabetesForm = ({title, description, options}: DiabetesFormProps) => {
```

Comment:
Id rename this to be a general name and not specific to the diabetes type as we want this to be as reusable as possible. Something like ```export const RadioGroup```. Keep in mind changing this will require code changes in other places as well.